doc/dev/sharp_bits.rst

```diff
@@ -1064,6 +1105,9 @@ supported:
 >>> g(a, b)
 ValueError: Incompatible shapes for broadcasting: shapes=[(1, Traced<ShapedArray(int64[], weak_type=True)>with<DynamicJaxprTrace(level=3/0)>), (1, Traced<ShapedArray(int64[], weak_type=True)>with<DynamicJaxprTrace(level=3/0)>)]
```

Comment:
This error is not great because it doesn't explain what the issue is :(

Reply:
agree